src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2341,19 +2363,15 @@ public override BoundNode VisitAssignmentOperator(BoundAssignmentOperator node)
         {
             Debug.Assert(!IsConditionalState);
 
-            VisitLvalue(node.Left);
-            Result left = _result;
-
-            VisitRvalue(node.Right);
-            Result right = _result;
+            var left = node.Left;
+            VisitLvalue(left);
+            Result leftResult = _result;
 
-            // byref assignment is also a potential write
-            if (node.IsRef)
```

Comment:
>if (node.IsRef) [](start = 12, length = 15)

Are we losing special handling for ref assignments?

Reply:
It's not clear where this comment is attached but if it refers to calling `WriteArgument` for `ref` variables, there is no need since `WriteArgument` is a no-op in `NullableWalker`.

---
In reply to: [183864605](https://github.com/dotnet/roslyn/pull/26356#discussion_r183864605) [](ancestors = 183864605)